--- tokio-tls-0.2.1/Cargo.toml	1970-01-01T00:00:00+00:00
+++ tokio-tls-0.2.1/Cargo.toml	2019-06-22T12:51:20.776222+00:00
@@ -39,13 +39,5 @@
 version = "0.1"
 [target."cfg(all(not(target_os = \"macos\"), not(windows), not(target_os = \"ios\")))".dev-dependencies.openssl]
 version = "0.10"
-[target."cfg(any(target_os = \"macos\", target_os = \"ios\"))".dev-dependencies.security-framework]
-version = "0.2"
-[target."cfg(windows)".dev-dependencies.schannel]
-version = "0.1"
-
-[target."cfg(windows)".dev-dependencies.winapi]
-version = "0.3"
-features = ["lmcons", "basetsd", "minwinbase", "minwindef", "ntdef", "sysinfoapi", "timezoneapi", "wincrypt", "winerror"]
 [badges.travis-ci]
 repository = "tokio-rs/tokio-tls"
